src/components/utils/onfidoApi.ts

```diff
@@ -303,7 +305,68 @@ export const requestChallenges = (
   performHttpReq(options, onSuccess, (request) => formatError(request, onError))
 }
 
-/* v4 APIs */
+export const getApplicantConsents = (
+  applicantUUID: string,
+  url?: string,
+  token?: string
+) => {
+  if (!url) {
+    throw new Error('onfido_api_url not provided')
```

Comment:
should we add error messages to translation?

Reply:
If we don't have `url` here, it means that there's an issue with the integration. So it's more a technical error for the developer.